Core/Resources/ErrorMessages.resx

```diff
@@ -216,4 +216,7 @@
   <data name="AddingCarNotAllowed" xml:space="preserve">
     <value>Car with one or more of such properties already exists.</value>
   </data>
-</root>
+  <data name="TripIsAlreadyExistsInTheTimeSpace" xml:space="preserve">
+    <value>There is already a trip in this time space!</value>
+  </data>
+</root>
```

Comment:
This is the code of the table and it is not possible to insert an empty row at the end because there is no access to this file